### PR TITLE
Endpoint to get AWS Accounts status

### PR DESCRIPTION
**Pull request by hug33k**
This PR adds a new endpoint (`/aws/status`) that returns every users' AWS Account with their bill repositories and also their statuses.

This endpoint cas be used to check if there is an issue with AWS Account or bill repositories configuration.

### CODE REVIEW

File: aws/routes/routeAwsStatus.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+		)
+		res[id] = item
+		if err != nil {
+			return nil, err
+		}
+	}
+	return res, nil
+}
+
+func getAwsAccountsStatus(r *http.Request, a routes.Arguments) (int, interface{}) {
```
Comment (thibautcornolti) [inline aws/routes/routeAwsStatus.go]: You should keep your types. `(int, []awsAccountWithStatus, error)` would be better.

File: aws/routes/routeAwsStatus.go [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+		}
+	} else {
+		return status{
+			Value: "ok",
+			Detail: "",
+		}
+	}
+}
+
+func getLastJobs(tx *sql.Tx, billRepositoriesIds []int) (map[int]job, error) {
```
Comment (thibautcornolti) [inline aws/routes/routeAwsStatus.go]: Any heavy SQL code should be located in `models/awsbillupdatejob.go`.

File: aws/routes/routeAwsStatus.go [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+	var sqlstr = `
```
Comment (thibautcornolti) [inline aws/routes/routeAwsStatus.go]: You should create a view and, if you can, get values with XO.

**Review by thibautcornolti — CHANGES_REQUESTED** [review]

File: aws/routes/routeAwsStatus.go [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+		for _, billRepository := range awsAccount.BillRepositories {
+			billRepositoriesIds = append(billRepositoriesIds, billRepository.Id)
+		}
+	}
+	jobs, err := getLastJobs(tx, billRepositoriesIds)
+	if err != nil {
+		l.Error("failed to get AWS accounts' bill repositories import statuses", err.Error())
+		return 500, errors.New("failed to retrieve import statuses")
+	}
+	result := make([]interface{}, 0)
```
Comment (thibautcornolti) [inline aws/routes/routeAwsStatus.go]: No need to use `interface` here.

**Review by zanchi-r — APPROVED** [review]

**Pull request merged**